UnitTests/OIDAuthorizationRequestTests.m

```diff
@@ -69,6 +69,14 @@
  */
 static NSString *const kTestAdditionalParameterValue = @"1";
 
+/*! @brief Test key for the @c additionalHeaders property.
```

Comment:
If we decide to drop the changes to `OIDAuthorizationRequest`, then we drop changes here as well.

Reply:
I agree as well. I reverted the changes like in `OIDAuthorizationRequest`.